eoc_journal/translations/config.yaml

```diff
@@ -12,9 +12,6 @@ dummy_locales:
 # Directories we don't search for strings.
 ignore_dirs:
     - tests
-    - eoc_journal/conf
-    - eoc_journal/translations
-    - eoc_journal/public/js/translations
```

Comment:
Why are these being removed?

Reply:
reverted this change, perhaps not needed at this time.